source/includes/document-filtering-limitations.rst

```diff
@@ -0,0 +1,27 @@
+- ``field`` names can't contain dots (``.``) or dollar signs (``$``).
+
+- ``matchValues`` array items can't be any of the following BSON types: 
+
+  - Arrays 
+  - Embedded documents 
+  - Regular expressions
+
+- Documents must not move in or out of the filter during migration. If a source 
+  document from a prior sync no longer matches the query filter, 
+  ``mongosync`` deletes the document from the destination cluster.
```

Comment:
Where did this come from? I don’t know what this refers to.

It sounds like this is just referring to mongosync’s normal behavior of requiring the destination to be empty … ? Maybe a destinationDataHandling setting that deletes preexisting destination data?

There’s nothing in mongosync that specifically deletes things that mismatch the filter.

Reply:
I think Felipe meant to just remove the part about deleting things that mismatch the filter. We should still keep:

```
Documents must not move in or out of the filter during migration.
```

Reply:
Gotcha. Updating.

Reply:
NB: I suspect the PD varies substantially from the feature as we eventually implemented it. The project was sidelined for a bit, then revived with a reduced scope.